mkdocs/docs/getting-started/install.md

```diff
@@ -26,7 +26,7 @@ curl -s https://raw.githubusercontent.com/iter8-tools/iter8-install/main/install
     set -e
```

Comment:
@kalantar If you agree, I am in favor of removing the ??? info "Look inside install.sh" tooltip altogether in  install.md. It is more work for us in terms of ensuring consistency/correctness of the documentation

Reply:
agree. done.